### DISCUSSION TITLE
网站提供的那个饥荒版本。。。就是id小鸡 大神提供的那个版本，怎么安装mod啊。。。。。

### DISCUSSION THREAD

**Discussion by BXQC** [Gamuxorg/bbs#343] [游戏或网站运行反馈]
网站提供的那个饥荒版本。。。就是id小鸡 大神提供的那个版本，怎么安装mod啊。。。。。

**Comment by shouhuanxiaoji** [comment]
想玩mod的话就需要解包游戏
./xxx.appimage --appimage-extract
解包后就可以安装mod了

**Comment by BXQC** [comment]
谢了

**Comment by BXQC** [comment]
解压后启动应用程序出错怎么办。没解压能正常运行。。。

**Comment by shouhuanxiaoji** [comment]
@BXQC 请注意，网站已经更新了一个新版本的饥荒，你可以试试
